Local dev enhancements (#45)
phasehq/console: CONTRIBUTING.md

Any third party components incorporated into our code are licensed under the original license provided by the applicable component owner.

## Setup local development environment
### Dev server with hot reload
1. Create a **.env.dev** file using
    ```
    cp .env.dev.example .env.dev
    ```
2. Add atleast one OAuth provider in your **.env.dev**. Follow the [docs](https://docs.phase.dev/self-hosting/configuration/envars)

> 2. Check **dev-docker-compose.yml** file is populating correctly with env variables by running 

Reviewer: ```suggestion
2. Verify that **dev-docker-compose.yml** file is populated with the correct environment variables by running 
```

    ```
    docker compose -f dev-docker-compose.yml --env-file .env.dev config
    ```

> 3. If all the configs are correct, start the cotainers using 

Reviewer: ```suggestion
3. Start the containers using 
```

    ```
    docker-compose -f dev-docker-compose.yml --env-file .env.dev up
    ```

> 4. The Console is now running at <http://localhost> with HMR(Hot Module Reload)
>    You can also connect using https protocol - <https://localhost>
>    >**Note :** It might show **Your connection isn't private**. 
>    You can ignore the message and continue using localhost

Reviewer: ```suggestion
4. The Console is now running at <https://localhost> with HMR(Hot Module Reload) and a self-signed certificate.
   >**Note : Your browser might warn you about the self-signed certificate. You can safely accept the certificate and proceed. 
```

Updated this instruction to to use `https` by default, as some browsers will automatically upgrade you anyway. Also made the instruction for swallowing the self-signed cert more generic, as developers might be using different browser which display this warning differently. 

Also, @Jay-Iam lets update this to use `http` once #33  is resolved
